app/models/user.rb

```diff
@@ -26,6 +26,11 @@ class User < ActiveRecord::Base
   has_many :images, :foreign_key => :uid
   has_many :drupal_node, :foreign_key => 'uid'
   has_many :user_tags, :foreign_key => 'uid', :dependent => :destroy
+  has_many :active_relationships, class_name: 'Relationship', foreign_key: 'follower_id', dependent: :destroy
```

Comment:
So instead of a two tables for following and followers, I use a single table which 2 instances, active_relationship, passive_relationship.
Active relationship instance exists if user A follows user B
Passive relationship instance exists if user B follows user A